app/states/texas/validators/texas_settings.py

```
@@ -1,5 +1,8 @@
-from pydantic import ConfigDict, model_validator
-from sqlmodel import SQLModel
+from dataclasses import Field
+from typing import Optional
+from datetime import date
+from pydantic import ConfigDict, model_validator, BaseModel, BeforeValidator
+from sqlmodel import SQLModel, Field
```

Ruff (0.14.7)

Redefinition of unused `Field` from line 1

(F811)